workbench-core/example/infrastructure/integration-tests/tests/datasets/accessPermissions.test.ts

```diff
@@ -315,4 +314,171 @@ describe('DataSets access permissions integration tests', () => {
       });
     });
   });
+
+  describe('removeDataSetAccessPermissions', () => {
+    it('removes a read-only access permission.', async () => {
+      const createDataSetResponse = await adminSession.resources.datasets.create({}, true);
+      const dataSetId: string = createDataSetResponse.data.id;
+      const createGroupResponse = await adminSession.resources.groups.create({}, true);
+      const { groupId } = createGroupResponse.data;
+      await (adminSession.resources.datasets.children.get(dataSetId) as Dataset).addAccess({
+        permission: {
+          identityType: 'GROUP',
+          identity: groupId,
+          accessLevel: 'read-only'
+        }
```

Comment:
could we extract this into a const to DRY this test up a bit?

Reply:
groupId changes per test. It is not a constant.

Reply:
I mean within the test. It looks like the `permission` object is repeated a couple times

Reply:
I meant that it is `groupId` here, but it is `userId` in another block you highlighted.

Reply:
My suggestion was to do something like this
```
const permission = {
          identityType: 'GROUP',
          identity: groupId,
          accessLevel: 'read-only'
        }

...

...addAccess({permission});

...

...removeAccess({permission})

...toMatchObject({
        data: {
          dataSetId: dataSetId,
          permissions: [permission]
        }
      })


```

Reply:
is it blocking?

Reply:
No, just clarifying since there was a misunderstanding of my original comments